test/DeploymentHelper.test.ts

```diff
@@ -0,0 +1,222 @@
+  const communityFundsTarget = "0x" + "42".repeat(3).padEnd(38, "0") + "01";
+  const bridgedCowTokenAddress = "0x" + "26".repeat(20);
+  const foreignToken = "0x" + "13".repeat(20);
+  const gnoPrice = 42;
+  const nativeTokenPrice = 1337;
+  let deploymentParams: DeploymentHelperDeployParams;
+
+  beforeEach(async () => {
+    gnoToken = await waffle.deployMockContract(deployer, IERC20.abi);
+    wrappedNativeToken = await waffle.deployMockContract(deployer, IERC20.abi);
+    MultiTokenMediatorHome = await artifacts.readArtifact(
+      "MultiTokenMediatorInterface",
+    );
+    multiTokenMediatorHome = await waffle.deployMockContract(
+      deployer,
+      MultiTokenMediatorHome.abi,
+    );
+    DeploymentHelper = (
+      await ethers.getContractFactory(ContractName.DeploymentHelper)
+    ).connect(deployer);
+    deploymentParams = {
+      foreignToken,
+      merkleRoot,
+      multiTokenMediatorHome: multiTokenMediatorHome.address,
+      gnoToken: gnoToken.address,
+      wrappedNativeToken: wrappedNativeToken.address,
+      communityFundsTarget,
+      gnoPrice,
+      nativeTokenPrice,
+    };
+  });
+
+  describe("constructor parameters", async function () {
+    beforeEach(async function () {
+      await multiTokenMediatorHome.mock.bridgedTokenAddress.returns(
+        bridgedCowTokenAddress,
+      );
+      deploymentHelper = await DeploymentHelper.deploy(
+        ...constructorInput(ContractName.DeploymentHelper, deploymentParams),
+      );
+    });
+
+    it("has expected merkle root", async () => {
+      expect(await deploymentHelper.merkleRoot()).to.equal(merkleRoot);
+    });
+
+    it("has expected multiTokenMediator", async () => {
+      expect(await deploymentHelper.multiTokenMediator()).to.equal(
+        multiTokenMediatorHome.address,
+      );
+    });
+
+    it("has expected real deploymentHelper", async () => {
+      expect(await deploymentHelper.foreignToken()).to.equal(foreignToken);
+    });
+
+    it("sets the expected community funds target", async function () {
+      expect(await deploymentHelper.communityFundsTarget()).to.equal(
+        communityFundsTarget,
+      );
+    });
+
+    it("sets the expected GNO deploymentHelper", async function () {
+      expect(await deploymentHelper.gnoToken()).to.equal(gnoToken.address);
+    });
+
+    it("sets the expected GNO price", async function () {
+      expect(await deploymentHelper.gnoPrice()).to.equal(gnoPrice);
+    });
+
+    it("sets the expected wrapped native deploymentHelper", async function () {
+      expect(await deploymentHelper.wrappedNativeToken()).to.equal(
+        wrappedNativeToken.address,
+      );
+    });
+
+    it("sets the expected native deploymentHelper price", async function () {
+      expect(await deploymentHelper.nativeTokenPrice()).to.equal(
+        nativeTokenPrice,
+      );
+    });
+  });
+
+  describe("deploy", async function () {
+    it("reverts if the bridge contract does not yet exists", async () => {
```

Comment:
Nit:
```suggestion
    it("reverts if the bridged token exists", async () => {
```

Reply:
I don't understand this one.

Reply:
Oh, this was meant to fix only the `d` which was a mini typo, like this:
```suggestion
    it("reverts if the bridged contract does not yet exists", async () => {
```

For whatever reason I changed more than I should have...